### PR TITLE
Tweaks suicide to be a little more personal.

### PR DESCRIPTION
**Pull request by SweptWasTaken**
The previous suicide notifier looked like this
image

Now it looks like this 
image

I will add doAfter to the suicide thingy but it wouldn't work so i'll fix it when I wake up.

Also 

I think PopupMessageOtherClientsInRange should be moved to ISharedNotifyManager.cs

### CODE REVIEW

**Comment by RemieRichards** [comment]
You do know this is only *one* suicide right? there are more.
Are the rest fine?

File: Content.Server/GameObjects/Components/Interactable/WelderComponent.cs [hunk 1]
```diff
@@ -251,22 +259,50 @@ public void OnUpdate(float frameTime)
...
             {
                 PlaySoundCollection(WeldSoundCollection);
-                chat.EntityMe(victim, Loc.GetString("welds {0:their} every orifice closed! It looks like {0:theyre} trying to commit suicide!", victim));
+                PopupMessageOtherClientsInRange(victim, Loc.GetString("{0:theName} welds {0:their} every orifice closed! It looks like {0:theyre} trying to commit suicide!", victim), 15);
+                _notifyManager.PopupMessage(victim, victim, Loc.GetString("You weld your every orific closed!", Owner));
                 return SuicideKind.Heat;
             }
 
-            chat.EntityMe(victim, Loc.GetString("bashes {0:themselves} with the {1}!", victim, Owner.Name));
+            PopupMessageOtherClientsInRange(victim, Loc.GetString("{0:theName} bashes {0:themselves} with the {1}!", victim), 15);
```
Comment (exp111) [inline Content.Server/GameObjects/Components/Interactable/WelderComponent.cs:282]: The Loc String doesn't have a second parameter

File: Content.Server/GameObjects/Components/Interactable/WelderComponent.cs [hunk 2]
```diff
@@ -251,22 +259,50 @@ public void OnUpdate(float frameTime)
...
+            };
+
+            var result = await _doAfterSystem.DoAfter(doAfterEventArgs);
+
+            if (result != DoAfterStatus.Cancelled && TryWeld(5, victim, silent: true))
             {
                 PlaySoundCollection(WeldSoundCollection);
-                chat.EntityMe(victim, Loc.GetString("welds {0:their} every orifice closed! It looks like {0:theyre} trying to commit suicide!", victim));
+                PopupMessageOtherClientsInRange(victim, Loc.GetString("{0:theName} welds {0:their} every orifice closed! It looks like {0:theyre} trying to commit suicide!", victim), 15);
+                _notifyManager.PopupMessage(victim, victim, Loc.GetString("You weld your every orific closed!", Owner));
```
Comment (exp111) [inline Content.Server/GameObjects/Components/Interactable/WelderComponent.cs:278]: Why the `, Owner`?

**Comment by SweptWasTaken** [comment]
Yeah ofc I know there are more, I forgot to say that I was gonna do them tomorrow since it's 1am here.

File: Content.Server/GameObjects/Components/Interactable/WelderComponent.cs [hunk 3]
```diff
@@ -251,22 +259,50 @@ public void OnUpdate(float frameTime)
...
                 return SuicideKind.Heat;
             }
 
-            chat.EntityMe(victim, Loc.GetString("bashes {0:themselves} with the {1}!", victim, Owner.Name));
+            PopupMessageOtherClientsInRange(victim, Loc.GetString("{0:theName} bashes {0:themselves} with the {1}!", victim), 15);
+            _notifyManager.PopupMessage(victim, victim, Loc.GetString("You bash yourself with the {1}!", Owner));
```
Comment (exp111) [inline Content.Server/GameObjects/Components/Interactable/WelderComponent.cs:283]: It should be `{0}`. And i think `Owner.Name`

File: Content.Server/GameObjects/Components/Interactable/WelderComponent.cs [hunk 4]
```diff
@@ -256,17 +261,36 @@ public SuicideKind Suicide(IEntity victim, IChatManager chat)
...
                 PlaySoundCollection(WeldSoundCollection);
-                chat.EntityMe(victim, Loc.GetString("welds {0:their} every orifice closed! It looks like {0:theyre} trying to commit suicide!", victim));
+                PopupMessageOtherClientsInRange(victim, Loc.GetString("{0:theName} welds {0:their} every orifice closed! It looks like {0:theyre} trying to commit suicide!", victim), 15);
+                _notifyManager.PopupMessage(victim, victim, Loc.GetString("You weld your every orific closed!"));
                 return SuicideKind.Heat;
             }
 
-            chat.EntityMe(victim, Loc.GetString("bashes {0:themselves} with the {1}!", victim, Owner.Name));
+            PopupMessageOtherClientsInRange(victim, Loc.GetString("{0:theName} bashes {0:themselves} with the {0}!", victim), 15);
+            _notifyManager.PopupMessage(victim, victim, Loc.GetString("You bash yourself with the {0}!"));
```
Comment (exp111) [inline Content.Server/GameObjects/Components/Interactable/WelderComponent.cs:270]: Missing Parameter

**Comment by SweptWasTaken** [comment]
I'm not gonna add do_after to this as it would require a pretty significant refactor of the suicide component which I feel is outside of my scope. It's ready for a merge though.

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**